[2022-03-18 15:22:09,729] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: pbp_bucket_ingestion_dag.get_pbp_data scheduled__2011-01-17T04:00:00+00:00 [queued]>
[2022-03-18 15:22:09,769] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: pbp_bucket_ingestion_dag.get_pbp_data scheduled__2011-01-17T04:00:00+00:00 [queued]>
[2022-03-18 15:22:09,770] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-03-18 15:22:09,772] {taskinstance.py:1239} INFO - Starting attempt 2 of 3
[2022-03-18 15:22:09,774] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-03-18 15:22:09,800] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): get_pbp_data> on 2011-01-17 04:00:00+00:00
[2022-03-18 15:22:09,812] {standard_task_runner.py:52} INFO - Started process 421 to run task
[2022-03-18 15:22:09,826] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'pbp_bucket_ingestion_dag', 'get_pbp_data', 'scheduled__2011-01-17T04:00:00+00:00', '--job-id', '6455', '--raw', '--subdir', 'DAGS_FOLDER/pbp_ingestion_dag.py', '--cfg-path', '/tmp/tmpam3bpmgr', '--error-file', '/tmp/tmp66h9lyuh']
[2022-03-18 15:22:09,834] {standard_task_runner.py:77} INFO - Job 6455: Subtask get_pbp_data
[2022-03-18 15:22:10,109] {logging_mixin.py:109} INFO - Running <TaskInstance: pbp_bucket_ingestion_dag.get_pbp_data scheduled__2011-01-17T04:00:00+00:00 [running]> on host 9a1698cd207d
[2022-03-18 15:22:10,285] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=pbp_bucket_ingestion_dag
AIRFLOW_CTX_TASK_ID=get_pbp_data
AIRFLOW_CTX_EXECUTION_DATE=2011-01-17T04:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2011-01-17T04:00:00+00:00
[2022-03-18 15:22:10,633] {python.py:175} INFO - Done. Returned value was: None
[2022-03-18 15:22:10,665] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=pbp_bucket_ingestion_dag, task_id=get_pbp_data, execution_date=20110117T040000, start_date=20220318T152209, end_date=20220318T152210
[2022-03-18 15:22:10,736] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-03-18 15:22:10,820] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
